openapi/python-asyncio.sh

```diff
@@ -82,6 +82,9 @@ if [ ${PACKAGE_NAME} == "client" ]; then
   find "${OUTPUT_DIR}/client/" -type f -name \*.py -exec sed -i "s/async parameter/async_req parameter/g" {} +
   find "${OUTPUT_DIR}/client/" -type f -name \*.py -exec sed -i "s/if not async/if not async_req/g" {} +
 
+  # workaround https://github.com/swagger-api/swagger-codegen/pull/8061 (thread pool only on demand)
```

Comment:
Can you add `TODO: Remove this when above merges` (and below too)

Reply:
Sure, added.